server/Arcadia.Assistant.CSP/Vacations/CspEmployeeVacationsRegistry.cs

```diff
@@ -47,7 +50,7 @@ protected override void OnReceive(object message)
                     break;
 
                 case Initialization.Success msg:
-                    foreach (var @event in msg.Events.Where(e => e.CalendarEvent.IsPending))
+                    foreach (var @event in msg.Events)
```

Comment:
intended?

Reply:
Yes. Some actors want to know about only pending events recovery, but reminder wants to know about approved events. That's why I moved this logic to concrete receiver of event bus message.